scripts/migrate.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+(async () => {
+  const {promisify} = require('util');
+  const {readdir} = require('fs');
+  const path = require('path');
+  const { createClient } = require('contentful-management');
+  const {default: runMigration} = require('contentful-migration/built/bin/cli');
+
+  // utility fns
+  const getVersionOfFile = (file) => file.replace('.js', '').replace(/_/g, '.');
+  const getFileOfVersion = (version) => version.replace(/\./g, '_') + '.js';
+
+  //
+  // Configuration variables
+  //
+  const [,, SPACE_ID, ENVIRONMENT_ID, CMA_ACCESS_TOKEN] = process.argv;
+  const LOCALE = 'en-US';
```

Comment:
Not necessary for the demo but if we decide to split this off we should read the default locale from the space. (We're already getting the space in line 24)